docs/sources/flow/config-language/syntax.md

```diff
@@ -75,6 +75,21 @@ BLOCK_NAME "BLOCK_LABEL" {
 }
 ```
 
+#### Patter for creating repeated blocks
```

Comment:
Instead of saying this is a new pattern, can we add an extra sentence to the blocks header?

Something like:

> Some blocks support being defined more than once. 

I'm not sure it makes to document this with an example explicitly.